desktop/libs/notebook/src/notebook/connectors/sql_alchemy_tests.py

```diff
@@ -327,7 +327,7 @@ def test_dialect_trim_statement_semicolon(self):
           # Trim
           engine = SqlAlchemyApi(self.user, interpreter).execute(notebook, snippet)
 
-          execute.assert_called_with('SELECT 1')
+          execute.assert_called_with('SELECT 1', async_=True)
```

Comment:
Could we add separate tests for when async is True instead of losing the current behavior of the current ones? (e.g. we can assume only for when using pyhive for now)